src/components/Login.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+
+    <div class="container">
+        <h1>Login für Ladenbesitzer</h1>
+        <form class="wb-3">
+            <!-- <input type="text" name="usernameField" v-model="input.username" placeholder="Username" />
+            <input type="password" name="passwordField" v-model="input.password" placeholder="Password" /> -->
+            <div class="form-group mb-3" style="text-align: left; margin: 0 auto;">
+                <label for="usernameField">Username</label>
+                <input id="usernameField" class="form-control" v-model.trim="input.username" type="text" placeholder="Username">
+            </div>
+            <div class="form-group mb-3" style="text-align: left; margin: 0 auto;">
+                <label for="passwordField">Password</label>
+                <input id="passwordField" class="form-control" v-model.trim="input.password" type="password" placeholder="Password">
+            </div>
+            <button type="button" class="btn btn-primary" v-on:click="login()">
+                Login
+            </button>
+        </form>
+    </div>
+
+</template>
+
+
+<script>
+export default {
+    name: 'RegistrationLogin',
+    props: {
+        input: {
+            type: Object,
+            default: function() {
+                return {
+                    username: "",
+                    password: ""
+                }
+            }
+        }
+    },
+    methods: {
+        login() {
+            // Routes to internal area if login successful
+            if(this.input.username != '' && this.input.password != '') {
```

Comment:
POST to `/api/login` using ajax and check return code (200 = authenticated, 401 not authenticated).
It also provides info what the reason for the denial is (wrong_credentials or a missing_role.)

Reply:
```
import JQuery from 'jquery'
JQuery.ajax(
    '/api/login',
    {
         method: "POST",
         data: {username: this.username, password: this.password},
         success: () => {handle sucess case here},
         error: (response) => {handle error case here}
    }
)
```